### PR TITLE
Change in Resources files to follow the same convention throughout

### LINKED ISSUE #898: Make mentioned changes in resources files to follow the same convention
## Description
As a contributor,
I need consistent style followed in resources file,
so that I don't get confused that whether some part of the code is missing or not.

## Mocks
https://github.com/anitab-org/mentorship-backend/blob/6fe6929fed2007ed43242f36c44f9bcca1d79979/app/api/resources/user.py#L171-L182
Here the route is below the responses which breaks the convention.
https://github.com/anitab-org/mentorship-backend/blob/6fe6929fed2007ed43242f36c44f9bcca1d79979/app/api/resources/user.py#L247-L268
Here the responses aren't attached with the route function but with the put().

### Update [Required]
- [ ] Make the above mentioned changes in the code to follow the same convention throughout the file.
- [ ] Check the same with other resources files and fix them if such convetion is broken.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
4 hours

### PR DESCRIPTION
**Pull request by gitpurva**
### Description

Same convention in resources files

Fixes #898 

### Type of Change:

- Code

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**

- [x] My changes generate no new warnings 
- [x] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by PrashanthPuneriya** [comment]
@gitpurva You should always create a new branch on the top of develop branch while working on a new issue.
@devkapilbansal Do you know any workaround/fix ?

**Review by devkapilbansal — CHANGES_REQUESTED** [review]
@gitpurva you should rebase your commits on upstream/develop
Also, it would be good is you squash your commits in one. It's always better to use a single commit to solve an issue.
Here is the workflow you should follow:
```
git remote add upstream https://github.com/anitab-org/mentorship-backend.git
git rebase upstream/develop
git reset --soft HEAD~6
git commit --amend
```

**Comment by gitpurva** [comment]
@devkapilbansal Ohkk.. I will do as said and will update the PR  .  Thank
You for your guidance:)

**Comment by PrashanthPuneriya** [comment]
@gitpurva Any progress?

**Review by vj-codes — DISMISSED** [review]
LGTM 🎉 
PR on separate branch ✔️

**Comment by vj-codes** [comment]
@devkapilbansal review this again please!

**Comment by devkapilbansal** [comment]
> @devkapilbansal review this again please!

@vj-codes I don't know whether the changes are as per convention or not and what are the conventions to follow. Pinging @PrashanthPuneriya to review

File: app/api/dao/mentorship_relation.py [hunk 1]
```diff
@@ -275,23 +275,23 @@ def cancel_relation(user_id: int, relation_id: int):
         """
```
Comment (PrashanthPuneriya) [inline app/api/dao/mentorship_relation.py]: This file shouldn't be part of the PR

**Review by PrashanthPuneriya — COMMENTED** [review]
`doc, route, response, parameters, expect, marshal_list_with` convention looks good to me. I am not sure about the convention tho it's just my thought.
Can we make all the above mentioned things appear above the class @gitpurva?

**Comment by gitpurva** [comment]
I did not understand why the run test are failing as the same changes which I have commited earlier (which  passed all the test cases) but now it failed to do so inspite of the same changes which are pushed again:(
In the latest commits I haven't added anything extra in the file . I just placed all the methods above class to follow the convention.

**Comment by devkapilbansal** [comment]
@gitpurva try to amend your previous commit instead of making a new commit for pull request. It will help you to better understand the changes and which change might be causing build to fail. Also, it would help the maintainers keep the history clean

**Comment by vj-codes** [comment]
@gitpurva hey please solve the merge conflicts, thanks!

**Comment by devkapilbansal** [comment]
Hi @gitpurva why do you close this pull request. I thought that this pull request is very close to merging

**Comment by gitpurva** [comment]
Hello @devkapilbansal I have just undo the commits which I have committed(to fix the problem which I encountered by removing one file accidentally) and the pull request got closed automatically . I don't know how!:(  I am new to open source

**Comment by devkapilbansal** [comment]
Hi @gitpurva this is because there are no changes for which pull request is needed. You just push some changes to this branch and you will get an option to reopen this pull request

**Comment by devkapilbansal** [comment]
@gitpurva will you please reopen this pull request after making some commits or open a new PR??

**Pull request closed without merging**